build.gradle

```diff
@@ -167,22 +167,27 @@ eclipseClasspath {
     }
 }
 
+def isTravis = System.getenv("TRAVIS") != null
+def isAppVeyor = System.getenv("APPVEYOR") != null
+def isCi = isTravis || isAppVeyor
+
 task createConfigs {
     description "Sets up the project by obtaining necessary files and configurations points."
     group "Setup"
+    def testTemplateFilename = isCi ? "test.ci.properties" : "test.template.properties"
```

Comment:
Got it.
Actually, what is the purpose of separate `test.properties` template? Maybe we can work it out such that it is not needed.